docs/introvideos/quicktour.md

```diff
@@ -0,0 +1,35 @@
+---
+Order: 2
+Area: introvideos
+TOCTitle: Quick Tour using JavaScript
+ContentId: a51b707c-19f7-41af-b5c2-c296e6e9fea2
+PageTitle: Quick Tour of VS Code using JavaScript
+DateApproved: 10/03/2016
+MetaDescription: Quick Tour around VS Code using JavaScript.
+---
+
+# Quick Tour of VS Code using JavaScript
+
+What can you do with VS Code? This tutorial walks you through an overview of the core features of VS Code, including IntelliSense, debugging, Git version control integration, and more. 
```

Comment:
For first mention in a topic use "Visual Studio Code".
Maybe "This tutorial gives you an overview ..."